rust-toolchain.toml

```diff
@@ -0,0 +1,14 @@
+[toolchain]
+channel = "stable"
+components = [
+	"cargo",
+	"clippy",
+	"rust-analyzer",
+	"rust-src",
+	"rust-std",
+	"rustc",
+	"rustc-dev",
```

Comment:
why is `rustc-dev` required?

Reply:
Got it from here: https://github.com/paritytech/polkadot-sdk/blob/master/templates/solochain/env-setup/rust-toolchain.toml

Removed it because couldn't find a reason as well